NSS-Likeable-Lemurs/BangazonSite
# MGnavProdCat

## Link to Ticket
Closes # [Link](https://github.com/NSS-Likeable-Lemurs/BangazonSite/issues/3)

## Description of Proposed Changes
- Product Type link takes you to product type index page
- Displays all product types
- Displays qty of items in that type
- Product name is link to description page 
## Steps to Test

Outline the steps to test

```
git fetch --all
git checkout <MGnavProdCat>
```

1.Press play.
2.Press on Product Type.
3. See List of all Categories.
4. Click on product name to go to detail page

## Impacted Areas in Application

List general components of the application that this PR will affect:

* ProductTypesController
* ProductTypes Index.cshtml


## Mentions @username

@HelenChalmers 
@ThraxHamilton 
@streatorward 
@kellycook301 

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
